src/components/radios/index.njk

```diff
@@ -190,6 +190,20 @@ Let users select a single option from a list.
         html: 'Provide additional attributes to each radio item label. See `label` component for more details.'
       }
     ],
+    [
+      {
+        text: 'items.{}.divider'
+      },
+      {
+        text: 'text'
+      },
+      {
+        text: 'No'
+      },
+      {
+        html: 'Optional divider to separate options, e.g "or".'
```

Comment:
I think it needs to be clearer how to use this – it's not obvious for example that specifying 'divider' means that all other options will be ignored.